addons/web/tests/test_session_info.py

```diff
@@ -38,6 +38,8 @@ def setUpClass(cls):
     def test_session_info(self):
         """ Checks that the session_info['user_companies'] structure correspond to what is expected """
         self.authenticate(self.user.login, self.user_password)
+        # avoid the create part of res.users.settings since this route is readonly
+        self.env['res.users.settings']._find_or_create_for_user(self.user)
         response = self.url_open("/web/session/get_session_info", data=self.payload, headers=self.headers)
```

Comment:
Nitpicking: the comment refers to the `url_open` comment, so it's a bit confusing to read so it takes more time than necessary. Probably saying `get_session_info is readonly so let's create beforehand` would be less effort to understand these LOCs and the comment.